test/Misc/RelFinField.jl

```diff
@@ -1,5 +1,12 @@
 @testset "RelFinField" begin
 
+  @testset "conformance" begin
+    F = Native.finite_field(3, 3, cached = false)[1]
```

Comment:
@thofma I thought we'd rather get rid of the RelFinField?

Reply:
I only replaced the ones in the prime decomposition of relative number fields. They might be used somewhere else still.